gobblin-runtime/src/main/java/gobblin/runtime/instance/StandardGobblinInstanceDriver.java

```diff
@@ -333,6 +387,45 @@ public boolean isInstrumentationEnabled() {
     @Override public void switchMetricContext(MetricContext context) {
       throw new UnsupportedOperationException();
     }
+
+    public List<GobblinInstancePluginFactory> getDefaultPlugins() {
+      if (!getSysConfig().getConfig().hasPath(PLUGINS_FULL_KEY)) {
+        return Collections.emptyList();
+      }
+
+      String pluginsListStr = getSysConfig().getConfig().getString(PLUGINS_FULL_KEY);
+      List<String> pluginNames = Splitter.on(',').omitEmptyStrings().trimResults().splitToList(pluginsListStr);
```

Comment:
Have you looked at `ConfigUtils.getStringList(Config config, String path)` ? It supports both java props style comma separated strings and typesafe style list.

```
Examples below will all return a list [1,2,3] without quotes
a.b=[1,2,3]
a.b=["1","2","3"]
a.b=1,2,3
a.b="1","2","3"
```